portal/components/react-admin/base/resources/corporate-devices.js

```diff
@@ -395,13 +387,9 @@ export const CorporateDevicesEdit = (props) => {
     [mutate]
   );
 
-  const sendOtp = async (phone_number) => {
+  const generateOtp = async (phone_number) => {
     if(phone_number && phone_number.length >= 10) {
-      const response = await axios({
-        method: "GET",
-        url: `${process.env.NEXT_PUBLIC_API_URL}/sendOTP?phone_number=${phone_number}`,
-      });
-      const responseObject = response.data;
+      const responseObject = await sendOTP(phone_number);
       if (!responseObject.error) {
         setOtpGenerate(true);
```

Comment:
Can we call this function `setOTPGenerated`

Reply:
Yes for OTP input read-only true false